PowerEditor/installer/nsisInclude/GetVersion.nsi

```diff
@@ -0,0 +1,70 @@
+!include "FileFunc.nsh"
+!include "logiclib.nsh"
+
+OutFile "GetVersion.exe"
+SetCompressor /SOLID lzma 		; Compression technique
+RequestExecutionLevel user 		; This is to make output exe run as invoker (Admin privileges not required)
+!define EXEPATH "..\bin\Notepad++.exe"	; Absolute path
+
+
+Section "Main"
+	SetAutoClose true	; it simply writes version info in the text. So close immediatly after writing version info
+
+    DetailPrint "Getting version from ${EXEPATH}"
+    GetDllVersion "${EXEPATH}" $R0 $R1 	; Two values were read during compilation
+	
+	;${GetFileVersion} "${EXEPATH}" $R0
+	;DetailPrint "Another way to get version: $R0"
+
+    IntOp $R2 $R0 / 0x00010000		; $R2 now contains major version
+    IntOp $R3 $R0 & 0x0000FFFF		; $R3 now contains minor version
+    IntOp $R4 $R1 / 0x00010000		; $R4 now contains release
+    IntOp $R5 $R1 & 0x0000FFFF		; $R5 now contains build
+	
+	; if notepad++.exe version = 7.0.0.0
+	; 	$1 = 7.0.0.0 (File Version)
+	;	$2 = 7		 (Product Version)
+	;	$3 = 0		 (Minor Version)
+	
+	; if notepad++.exe version = 7.1.2.0
+	; 	$1 = 7.0.0.0 (File Version)
+	;	$2 = 7.1.2 	 (Product Version)
+	;	$3 = 12		 (Minor Version)
+	
```

Comment:
Product version and File version **are** the same (see [Notepad++ resource file](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/src/Notepad_plus.rc)).
But there's a difference in notation for the [`VERSION_DIGITALVALUE`](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/src/resource.h#L35) and the [`VERSION_VALUE`](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/src/resource.h#L34).
The property list of the Notepad++ v6.9.2.0 executable will show the digital value (i.e. comma-separated-digits) as dot-separated-digits (e.g. `6.9.2.0`) and the textual value will be visible as defined in the [resource file](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/src/resource.h#L34) i.e. `6.92`.